### PR TITLE
Support for private comments

### PR DESCRIPTION
**Pull request by michelcaradec**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #84 

## What is the new behavior?

Comments now support a "private" state.
By default, private comments:

- Are displayed with an `eye-closed` icon (see <https://code.visualstudio.com/api/references/icons-in-labels#icon-listing>). This icon can be changed with the setting `code-review.privateCommentIcon`.
- Are not exported. The export can be forced with the setting `code-review.reportWithPrivateComments`.

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

The extension now depends on VSCode v1.52 in order to use `ThemeColor` in `TreeItem` (see <https://code.visualstudio.com/updates/v1_51#_codicon-colors-in-trees>).

## Other information

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -239,6 +251,35 @@
...
+      },
+      {
+        "id": "codereview.priority.red",
+        "description": "Color for comments with priority level 3 in code-review extension",
+        "defaults": {
+          "dark": "#ff0000",
+          "light": "#ff0000",
+          "highContrast": "#ff0000"
+        }
+      }
```
Comment (d-koppenhagen) [inline package.json:282]: These settings should also be listed in the README.md

Comment (michelcaradec) [reply package.json:282]: Sorry, I forgot this one.
Done.

**Review by d-koppenhagen — APPROVED** [review]
lgtm

**Comment by d-koppenhagen** [comment]
I will release it later on today

**Pull request merged**